### ISSUE TITLE
Can't get this to work

### ISSUE THREAD

**Issue by slightlyfaulty** [tkkcc/libinput_patch#2]
Is this patch still functional? It doesn't seem to be doing anything on my end.

Running `Manjaro KDE` with `libinput 1.14.1`.

- Installed from AUR
- Restarted PC
- `echo 6 > /tmp/libinput_discrete_deltay_multiplier`
- Tested in Google Chrome and a few other apps, scroll speed remains the same.

Is there anything else I can do to debug? Would be great to get this working!

**Comment by linuswillner** [comment]
I installed this from AUR and it worked just fine out of the box. Did you remove `libinput` when prompted?

**Comment by slightlyfaulty** [comment]
@linuswillner Yup `libinput` seems to be removed automatically.

Does this maybe only work with `i3wm`? I'm using `kwin` on KDE Plasma and this just seems to do nothing at all.

**Comment by linuswillner** [comment]
I’m on KWin myself, so that’s probably not the issue. You can always try switching to i3wm and see if it helps, but I have my doubts. It does sound like either your installation is somehow broken, or the scroll speed multiplier isn’t in /tmp somehow.

**Comment by slightlyfaulty** [comment]
Thanks for the reply @linuswillner. I've tried everything I can think of to no avail. Gonna close this out as a dead end. Perhaps my Logitech G Pro Wireless just isn't compatible in some way or another.

**Comment by gammons** [comment]
@slightlyfaulty I've noticed that this does not work unless I `chown` the `/tmp/libinput_discrete_deltay_multiplier` to the user I am logged in as.  Otherwise it does not work.

**Comment by slightlyfaulty** [comment]
@gammons Thanks for the tip, but it still doesn't work for me unfortunately.

**Comment by slightlyfaulty** [comment]
@gammons Actually I just rebooted and it's working now. Thanks again!

**Comment by gammons** [comment]
It seems finicky TBH.  I'm using GDM and I3.  I've noticed that I need to log into I3, change the ownership of that file + update the number in it, then log out of i3 and log back in again to get it to work.   It seems like GDM creates the file, and the owner is `gdm`.  I'm assuming that's because libinput gets initialized somewhere already in that flow.

Perhaps if I did not use GDM, I'd be able to set this in a more automated way, but so far I haven't had good luck figuring out how to do that.
